manifests/lib/perl.pp

```diff
@@ -1,5 +1,10 @@
-# This class installs the perl libs for postgresql. See README.md for more
-# details.
+# @summary This class installs the perl libs for postgresql.
+#
+# @param package_name
+#   String. Specifies the name of the PostgreSQL perl package to install.
+# @param package_ensure
+#   Enum['present','absent'].
```

Comment:
Same as before. The data type itself needs changed to reflect this. Also needs description.